web-frontend/src/pages/TransferCoins/SendCoins/SendConfirmation.tsx

```diff
@@ -20,7 +20,28 @@ export function SendConfirmation({ ...props }) {
 
   const formattedTotal = formatStandard(toMASS(total));
   const [showTooltip, setShowTooltip] = useState(false);
-
+  const [customFees, setCustomFees] = useState<string>('Standard');
```

Comment:
put string 'Standard' into a const variable, same for Low, High and Custom. This will be the technical fee enum (so no uppercase).

Reply:
in addition the name `customFees` don't allow me to understand what this variable represents.

Reply:
would `selectedFees` be ok ?